src/main/java/tri/stringreverse/StringReverse.java

```diff
@@ -0,0 +1,16 @@
+package tri.stringreverse;
+
+/**
+ * Output a reversed string.
+ */
+public class StringReverse {                //create class
+    public String reverse(String text){     //function named reverse, the function used a built-in reverse function and return the reversed string
+        return reverse(text);
```

Comment:
With the infinite loop, the program's running but never give out a straight result, instead it keeps on running until the memory got overloaded.
I changed the return line with the line you suggested, and it ran well. 
Also, I found out that Java is case sensitive, I typed in Stringbuilder but it didn't accept it until I changed back to StringBuilder.